client/src/index.js

```diff
@@ -37,7 +38,9 @@ const store = customStore.create();
 ReactDOM.render(
   <div id='next-root'>
     <Provider store={store}>
-      <App routes={routes}/>
+      <BrowserRouter>
```

Comment:
Only one `BrowserRouter` should exist. This nests `<BrowserRouter>` in `App.js` within it. https://github.com/ReactTraining/react-router/issues/5291.